armi/reactor/converters/tests/test_uniformMesh.py

```diff
@@ -28,6 +28,29 @@
 from armi.tests import TEST_ROOT, ISOAA_PATH
 
 
+class DummyFluxOptions:
+    def __init__(self):
+        self.photons = False
+
+
+class TestConverterFactory(unittest.TestCase):
+    def setUp(self):
+        self.o, self.r = test_reactors.loadTestReactor(
+            inputFilePath=os.path.join(TEST_ROOT, "detailedAxialExpansion"),
+        )
+        self.dummyOptions = DummyFluxOptions()
+
+    def test_converterFactory(self):
+        print(self.dummyOptions)
```

Comment:
Can we remove this `print()` statement?  We try not to sprinkle `print`s throughout ARMI.

Reply:
ah yes, good catch